1209/A10C/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Aminoacid Synthesizer
+owner: KBLES
```

Comment:
I cannot change your pull request, only you can. If you're using the UI, you can click the 'edit' icon on the file; otherwise you need to make a new commit via the git CLI.